frontend/src/store/index.js

```diff
@@ -1660,6 +1676,41 @@ const actions = {
   setColorScheme ({ commit }, colorScheme) {
     commit('SET_COLOR_SCHEME', colorScheme)
     return state.colorScheme
+  },
+  async initializeClusterIdentity ({ commit, dispatch, state }) {
+    const { clusterIdentity } = state
+
+    if (!isEmpty(clusterIdentity)) {
+      // it is not expected that the cluster identity will ever change, hence we can just return it
+      return clusterIdentity
+    }
+
+    try {
+      const { data: { clusterIdentity } } = await getClusterIdentity()
+      commit('SET_CLUSTER_IDENTITY', clusterIdentity)
+
+      return clusterIdentity
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.warn(`Failed to get cluster-identity ${err.message}`)
+    }
+  },
+  refreshGardenctlOptions ({ commit }) {
```

Comment:
I don't get the idea behind this action. Is it a getter for the default `gardenctlOptions` or is it used to initialize the localStorage. Why don't we use the same approach as with `global/color-scheme` and initialize the store in the root app.